### PR TITLE
Fixes #14390: Missing report on generic method "command exec" with $()

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/14390

### CODE REVIEW

File: builder/js/ncf.js [hunk 1]
```diff
@@ -1149,7 +1159,30 @@ $scope.onImportFileChange = function (fileEl) {
           // We will lose the link between the selected method and the technique, to prevent unintended behavior, close the edit method panel
           $scope.selectedMethod = undefined;
         }
-        $scope.resetFlags();
+        $scope.resetFlags();i
```
Comment (VinceMacBuche) [inline builder/js/ncf.js]: There is a 'i' after the semi-colon here!!

Comment (amousset) [reply builder/js/ncf.js]: :eye: :eye_speech_bubble: :eyes: :bird:

Comment (ncharles) [reply builder/js/ncf.js]: JS is so forgiving

**Comment by ncharles** [comment]
Commit modified

File: builder/js/ncf.js (hunk 1 above)
Comment (VinceMacBuche) [reply builder/js/ncf.js]: i does nothing, you only call a variables that is undefined ... but yes ...

**Pull request merged**